tools/run_tests/sanity/banned_functions.py

```diff
@@ -50,11 +50,6 @@
     ],
     "gpr_log(": [
         "./include/grpc/support/log.h",
-        "./src/core/client_channel/client_channel.cc",
```

Comment:
Why is this diff needed?

Reply:
This is a list of files where use of gpr_log is allow - listed.
As we clean up the files, we keep deleting entries from here to prevent backsliding.

Reply:
This is a part of the gRPC sanity tests.